python-for-loop/upacking.py

```diff
@@ -0,0 +1,4 @@
+points = [(1, 4), (3, 6), (7, 3)]
+
+for x, y in points:
+    print(f"{x = } and {y = }")
```

Comment:
Same as in the previous comment, flake8 doesn't like the whitespace:

```suggestion
    print(f"{x=} and {y=}")
```